src/convertToObject.js

```diff
@@ -5,8 +5,26 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+
+function convertToObject(stylesString) {
+  const styleObject = {};
+
+  const declarations = stylesString
+    .split(';')
+    .filter((declaration) => declaration.trim());
+
+  declarations.forEach((declaration) => {
+    const [property, value] = declaration.split(':');
```

Comment:
Consider using `declaration.split(':').map(item => item.trim())` to trim both `property` and `value` in one step. This will simplify the code and ensure that both parts are trimmed before checking their existence.